src/pages/UserProfilePage.tsx

```diff
@@ -1,94 +1,102 @@
-import { useParams } from 'react-router-dom';
+import { useLocation, useParams } from 'react-router-dom';
 import { users } from '../mocks/users';
 import { convertGenderToLabel } from '../utils/convertGenderToLabel';
 import { calculateAge } from '../utils/calculateAge';
 import InfoFormField from '../components/MyPage/InfoFormField';
 import UserProfileBtn from '../components/UserProfileBtn';
+import ChatFloatingBtn from '../components/Chat/ChatFloatingBtn';
 
 const UserProfilePage = () => {
   const { userId, roomId } = useParams();
+  const location = useLocation();
+
+  const isChatUserProfilePage = location.pathname !== '/chat/profile';
```

Comment:
useLocation 사용해서 해당 주소에서 렌더링 될 때 버튼 조건부 렌더링하기 위해서 여기만 수정했어요.

Reply:
```suggestion
  const isChatUserProfilePage = location.pathname.includes('/chat/profile');
```

지금은 채팅창이 아닌곳에서 프로필페이지에 들어갔을 때 채팅버튼 2개가 렌더링되어서 겹치네요